layouts/partials/plugin/share.html

```diff
@@ -4,8 +4,8 @@
   {{- $title := title .Title -}}
   {{- /* 001: Twitter */ -}}
   {{- if $share.Twitter -}}
-    <a href="javascript:void(0);" title="{{ T `shareOn` }} Twitter" data-sharer="twitter" data-url="{{ .Permalink }}" data-title="{{ $title }}"{{ with .Site.Params.Social.Twitter }} data-via="{{ . }}"{{ end }}{{ with .Params.tags }} data-hashtags="{{ delimit . `,` }}"{{ end }}>
-      {{- dict "Class" "fa-brands fa-twitter fa-fw" | partial "plugin/icon.html" -}}
+    <a href="javascript:void(0);" title="{{ T `shareOn` }} X" data-sharer="twitter" data-url="{{ .Permalink }}" data-title="{{ $title }}"{{ with .Site.Params.Social.Twitter }} data-via="{{ . }}"{{ end }}{{ with .Params.tags }} data-hashtags="{{ delimit . `,` }}"{{ end }}>
+      {{- dict "Class" "fa-brands fa-x fa-fw" | partial "plugin/icon.html" -}}
```

Comment:
It should be fa-x-twitter

Reply:
Thanks! I have changed to use the correct icon name. 
I also do the same change to the social icon in homepage under personal profile.